### DISCUSSION TITLE
Regarding Post 5601 - createSigneUrl - 404 - clarification please.

### DISCUSSION THREAD

**Discussion by fadethegap** [supabase/supabase#6514] [Questions]
@GaryAustin1 
Here is the link to the post in question: [5601](https://github.com/supabase/supabase/discussions/5601)

Gary, 

In the useEffect, you passed 'image' in as a prop. 

In the body you reference <file>.

What is the distinction between the two?

**Comment by GaryAustin1** [comment]
I'm really not sure what this in regards to.  If you have a question for the author of that discussion, you might ask there.  That was their code not mine.
